src/View/StringTemplate.php

```diff
@@ -284,7 +306,7 @@ protected function _formatAttribute($key, $value, $escape = true)
         $truthy = [1, '1', true, 'true', $key];
         $isMinimized = isset($this->_compactAttributes[$key]);
         if ($isMinimized && in_array($value, $truthy, true)) {
-            return "$key=\"$key\"";
+            return "$key";
```

Comment:
Looks good so far. But why this is necessary? As far as I know both formats are valid (both short and long syntax).
And this makes a lot of other tests fails, as well.